_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,214 @@
+
+Once enabled and active, this service will examine the trust policies and access to the following resources:
+
+- Amazon Simple Storage Service buckets;
+- AWS Identity and Access Management roles;
+- AWS Key Management Service keys;
+- AWS Lambda functions and layers;
+- Amazon Simple Queue Service queues.
+
+The IAM Access Analyzer will scan only within the boundaries of the AWS Account or Organization it has been enabled for. The results from this scan will be visible and accessible through the AWS CLI and the AWS Web console. For more information and details on what the AWS IAM Access Analyzer can achieve for your AWS Account and Organization, please refer to the official https://docs.aws.amazon.com/IAM/latest/UserGuide/what-is-access-analyzer.html[AWS docs].
+
+To help you achieve CIS 1.3 compliance, you can find examples of how to use the ready-made module as part of the `landingzone` module in the `aws-service-catalog` repository https://github.com/gruntwork-io/aws-service-catalog/releases/tag/v0.15.4[v0.15.4], and specifically the `account-baseline-root` and `account-baseline-security` https://github.com/gruntwork-io/aws-service-catalog/tree/v0.15.4/examples/for-learning-and-testing/landingzone[examples].
+
+==== 3. Deploy the ebs-encryption-multi-region module (recommendation 2.2.1)
+
+EC2 supports encryption at rest when using the Elastic Block Store (EBS) service. While disabled by default, forcing encryption when creating EBS volumes is supported. Encrypting data at rest reduces the likelihood that it is
+unintentionally exposed and can nullify the impact of disclosure if the encryption remains unbroken. Recommendation 2.2.1 specifies a manual process to encrypt EBS volumes using the AWS Console, however Gruntwork has developed
+a module that configures volume encryption by default in all enabled regions. Check out the https://github.com/gruntwork-io/module-security/tree/master/modules/ebs-encryption-multi-region[ebs-encryption-multi-region] to
+configure AWS EBS encryption in all enabled regions of an AWS Account.
+
+==== 4. Deploy the vpc-app-network-acl and vpc-mgmt-network-acl modules (recommendation 5.1)
+
+If you are already using the `vpc-app-network-acl` and/or `vpc-mgmt-network-acl` modules, you need to update the
+source URL from the https://github.com/gruntwork-io/terraform-aws-vpc[`terraform-aws-vpc` module] (previously known
+as `module-vpc`). There is one new required argument, `allow_administrative_remote_access_cidrs`. You will add the
+CIDRs from your office!
+
+[source,hcl]
+----
+module "vpc_app_network_acls" {
+  source = "git::git@github.com:gruntwork-io/cis-compliance-aws.git//modules/vpc-app-network-acls?ref=v0.9.3"
+
+  # ... the existing variables weren't affected!
+
+  allow_administrative_remote_access_cidrs = {
+    berlin_office = "1.2.3.4/32"
+    ny_office     = "6.7.8.9/32"
+  }
```

Comment:
This example needs to be updated to use 3 input variables, one per subnet tier:

- `allow_administrative_remote_access_cidrs_public_subnets`
- `allow_administrative_remote_access_cidrs_private_app_subnets`
- `allow_administrative_remote_access_cidrs_private_persistence_subnets`